### DISCUSSION TITLE
Missing .NET MAUI in Visual Studio 2022 RC2

### DISCUSSION THREAD

**Discussion by EmanuelBergTelia** [dotnet/maui#3057] [Q&A] [answered]
Hi,

I could not find .NET MAUI in Visual Studio 2022 RC2, is this due to not shipping in november?

**Comment by TonyValenti** [comment]
Same issue here. Was hoping to try it out.

**Comment by acaliaro** [comment, marked as answer]
I think you have Maui only in Preview, not in RC

**Reply by maddymontaquila** [reply]
Yup, .NET MAUI will only be part of the installer in the Preview release of VS 2022 (17.1) which launches when VS 2022 goes GA

**Comment by maxkoshevoi** [comment]
Try installing the workload, maybe it will appear.
```powershell
dotnet workload install maui
```

Also running `maui-check` might help
```powershell
dotnet tool update -g  redth.net.maui.check
maui-check
```

**Reply by TonyValenti** [reply]
Is that an option in the vs 2022 installer?

**Reply by maxkoshevoi** [reply]
All of those are command line commands
